src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DacFxDeployRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
+{
+    /// <summary>
+    /// Parameters for a DacFx deploy request.
+    /// </summary>
+    public class DacFxDeployParams : IScriptableRequestParams
+    {
+        /// <summary>
+        /// Gets or sets connection string of the target database the deploy operation will run against.
+        /// </summary>
+        public string ConnectionString { get; set; }
```

Comment:
this application is generally using connection URIs that maintain connection details with the ConnectionService.  Is there a reason not to follow that pattern with these new classes?

Reply:
No I will remove ConnectionString and switch to using SqlConnection. I initially didn't see that I could get the connectionstring (which is needed to instantiate DacServices) from it and was just trying to get something working.